.github/workflows/release.yml

```
@@ -0,0 +1,246 @@
+
+        # Update version in composer.json
+        composer config version $VERSION
+
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        git add composer.json
+        git commit -m "chore: update version to $VERSION"
+        git push origin main
+
+    - name: Post to Slack (optional)
+      if: secrets.SLACK_WEBHOOK_URL != ''
```

actionlint (1.7.9)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)